### ISSUE TITLE
[Bug] Failed to install the latest version 1.22

### ISSUE THREAD

**Issue by Bugazelle** [chrismattmann/tika-python#249]
Hello Chris,

We are failed to install the latest 1.22 at Python2.7.
Would you please take time to fix that?

```
ERROR: Command errored out with exit status 1:
     command: /usr/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-gIkGdk/tika/setup.py'"'"'; __file__='"'"'/tmp/pip-install-gIkGdk/tika/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-gIkGdk/tika/pip-egg-info
         cwd: /tmp/pip-install-gIkGdk/tika/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-gIkGdk/tika/setup.py", line 37, in <module>
        with open(os.path.join(os.path.dirname(__file__), 'README.md'), encoding='utf-8') as f:
    TypeError: 'encoding' is an invalid keyword argument for this function
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

Thank you,
Ken

**Comment by chrismattmann** [comment]
fixed in 
```
pomodoro:tika-python mattmann$ git commit -m "Fix for #122: https://github.com/cjhutto/vaderSentiment/issues/47"
[master 8b88be2] Fix for #122: https://github.com/cjhutto/vaderSentiment/issues/47
 1 file changed, 1 insertion(+)
pomodoro:tika-python mattmann$ git push -u origin master
Enumerating objects: 5, done.
Counting objects: 100% (5/5), done.
Delta compression using up to 12 threads
Compressing objects: 100% (3/3), done.
Writing objects: 100% (3/3), 366 bytes | 366.00 KiB/s, done.
Total 3 (delta 2), reused 0 (delta 0)
remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
To github.com:chrismattmann/tika-python.git
   90590d8..8b88be2  master -> master
Branch 'master' set up to track remote branch 'master' from 'origin'.
pomodoro:tika-python mattmann$ 
```

**Comment by chrismattmann** [comment]
fixed in https://github.com/chrismattmann/tika-python/commit/8b88be2e731ad7d14e91b8475b2c15e70b0cca2c
